include/comp_vol.h

```diff
@@ -0,0 +1,302 @@
+// VolEsti
+
+// Copyright (c) 2012-2017 Vissarion Fisikopoulos
+
+// VolEsti is free software: you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or (at
+// your option) any later version.
+//
+// VolEsti is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+// for more details.
+//
+// See the file COPYING.LESSER for the text of the GNU Lesser General
+// Public License.  If you did not receive this file along with HeaDDaCHe,
+// see <http://www.gnu.org/licenses/>.
+
```

Comment:
Use "include guards" in all .h files. 
```
#ifndef FILE_PATH_H
#define FILE_PATH_H

/* ... Declarations etc here ... */

#endif
```
here the FILE_PATH_H is COMP_VOL_H

By the way maybe the name ```volume``` would be better for that file.